### ISSUE TITLE
iOS: Accessibility Actions

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30893]
## Description
[accessibilityScroll](https://developer.apple.com/documentation/objectivec/nsobject/1615161-accessibilityscroll?language=objc) is missing from the list of actions, which would allow a view to perform a scroll action (three finger swipe up or down. Used in carousels, lists, or collections.)

## React Native version:
v0.63
